docs/helper-functions.md

```diff
@@ -0,0 +1,64 @@
+## Helper Functions
+
+**`whitelist_html`**
+
+When translating strings in WordPress, the most common functions to use are
+`__()` (translate and return) or `_e()` (translate and output). Where possible,
```

Comment:
Bit annoying to do but you'll need to remove these line breaks - they can cause odd formatting in some markdown compilers eg. putting `<br>` tags in odd places. A paragraph should have no line breaks.